src/main/java/com/robertx22/age_of_exile/database/data/talent_tree/TalentTree.java

```diff
@@ -119,6 +123,25 @@ public void onLoadedFromJson() {
 
         grid.loadIntoTree();
 
+        CompletableFuture.runAsync(() -> {
```

Comment:
Is there a reason this is async? Sorry i'm not familiar with how this works. I can just remove the async part if it breaks?

Reply:
prob not, the handlePaintQueue() on my machine will cost about 1 minute, the game might be completely frozen if this task doesn't run async